README.md

```diff
@@ -1,8 +1,107 @@
+Table of Contents
+=================
 
+* [Introduction](#introduction)
+* [Charts repository](#repository)
+  * [Available Charts in Repository](#available-charts-in-repository)
+  * [Usage](#usage)
+  * [Development](#development)
+  * [Testing](#testing)
+* [Templates](#templates)
+  * [Role-Based Access Control](#rbac)
 
-## How to play
+# Introduction
+
+This repository contains Helm templates and charts used by other applications part of the Snowdrop team.
+
+# Repository
+
+The repository has been configured to serve the static helm index and chart files. To use it:
```

Comment:
`The repository has been configured` How ? Can you be more explicit ?

Reply:
Changed